frontend/src/metabase/entities/fields.js

```diff
@@ -91,7 +91,7 @@ const Fields = createEntity({
       return Fields.actions.update(
         { id: field.id },
         field,
-        undo(opts, field.display_name, t`updated`),
+        notify(opts, field.display_name, t`updated`),
```

Comment:
Change from allowing undoing to only notifying that the change has been made, due to undoing actions seems to cause more trouble than helping us.
https://github.com/metabase/metabase/pull/20915#pullrequestreview-904419004